### DISCUSSION TITLE
Active Seeders

### DISCUSSION THREAD

**Discussion by ycagel** [DigiByte-Core/digibyte#246] [General]
Who is committed to running a DigiByte seeder through the life cycle release of v8.22? It is important for the community to ensure the seeders are up & running for new wallets to connect to when 1st loading up a core wallet as the DGB community grows into the future.

**Comment by JaredTate** [comment]
For people's reference here are the current seeders being provided by community members located in the DGB core code:

https://github.com/DigiByte-Core/digibyte/blob/22528bf4b77d1952fe28da918e5e71a7be1efc22/src/chainparams.cpp#L154-L162

I can setup a new seeder to run on a community hosted server being donated for DigiHash & digibyte.io and it can continue to run under the digibyte.io domain name. 

My plan is to shut down my own personal server the digibyte.io seeder is on once 8.22 is released and I relinquish my github access to the DigiByte Core Github organization. But I will make sure its up and going before then.

Knowing who will keep providing seeders is important to finalize before an 8.22 release.

**Comment by JohnnyLawDGB** [comment]
I will continue to run a seeder whenever we transition to 8.22.

It is why I initially set one up and will be utilizing the same domain as I am presently.

**Comment by ycagel** [comment]
Any additional updates?

**Comment by saltedlolly** [comment]
Better to ask in DGBCIT. I am still running my mainnet and testnet seeders for the time being. I will reach out to DGBCIT if that changes.

**Reply by ycagel** [reply]
> Better to ask in DGBCIT. I am still running my mainnet and testnet seeders for the time being. I will reach out to DGBCIT if that changes.

I have posted there.

**Comment by nadav32p** [comment]
Hey all. I am downloading a core wallet and will be setting up a GPU miner.
I want to also set up a seeder but wanted to see if I'm overloading.
My computer uses: Processor AMD Ryzen 7 8845HS (3.80 GHz up to 5.10 GHz)
16gb RAM, 1TB ssd.

What do you think? Will I crash the computer?

(New computer so there is nothing on it)

**Reply by JohnnyLawDGB** [reply]
It is not necessarily process or memory intensive.  Does need to have a 100% uptime if possible, and is probably best accomplished through a VPS.

**Comment by Quakeitup** [comment]
Seed.Quakeguy.com will remain active.

**Reply by JohnnyLawDGB** [reply]
Bless.
